linera-rpc/src/grpc/client.rs

```diff
@@ -349,6 +347,12 @@ impl ValidatorNode for GrpcClient {
         Ok(client_delegate!(self, get_genesis_config_hash, req)?.try_into()?)
     }
 
+    #[instrument(target = "grpc_client", skip(self), err, fields(address = self.address))]
```

Comment:
Does that end up being a Prometheus metric or just a log?

Reply:
Just a log.

Reply:
Ok, I was thinking that it might be useful to track blobs' sizes.

Reply:
Probably, yes. We should consider adding that at some point.